### DISCUSSION TITLE
Unable to render rich display

### DISCUSSION THREAD

**Discussion by iam-anubhav** [devicons/devicon#1563] [General]
The icons are not visible when I am using in readme file. It shows unable to render rich display while opening the svg image in the icons folder of this repo. 
Tell me the solution for this.

![image](https://user-images.githubusercontent.com/57631674/205492005-3fee08ad-3d78-4e6b-82bd-854438ef8b76.png)

![image](https://user-images.githubusercontent.com/57631674/205492031-9bc1cdd2-64d3-4b3b-9df7-be82add67b03.png)

**Comment by PantheraRed** [comment]
Hello @iam-anubhav. I'm not sure what is causing this but they work flawlessly for me. Are you sure it's not an issue with your internet connection or GitHub client? Please give more info about this.

![image](https://user-images.githubusercontent.com/41539361/205498471-0fa03ae2-dd21-48a9-a547-26a8c28f2ea3.png)
![image](https://user-images.githubusercontent.com/41539361/205498915-4d8d6d89-2ff5-464e-8861-215186644665.png)
![image](https://user-images.githubusercontent.com/41539361/205498945-717bb75b-05ca-4cab-bedf-2ecf6df636d3.png)

**Comment by Snailedlt** [comment]
Works for me too. I'm guessing it's an issue with your browser or OS. Could also be some browser settings or an browser extension maybe?

**Comment by Snailedlt** [comment]
Closing this since it's likely not related to devicons, but a local issue. Feel free to comment if you find out it's a devicon issue, and I'll re-open

**Reply by Snailedlt** [reply]
Ohh wait, it's a discussion... Not an issue 😅

**Comment by iam-anubhav** [comment]
OK...Thanks @Snailedlt @PantheraRed for the help.
